examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -38,9 +60,38 @@
 ]
 class_mapping = dict(zip(range(len(class_ids)), class_ids))
 
+flags.DEFINE_string(
+    "weights_path",
+    "weights_{epoch:02d}.h5",
```

Comment:
It seems that this is not just a path, but a formatted string for the file name. Should we make it clear in the flag description?

Reply:
Changing to `weights_name`, compare to file_name